### DISCUSSION TITLE
如何自定义首页的md文件路径

### DISCUSSION THREAD

**Discussion by LasyIsLazy** [vuepress-reco/vuepress-theme-reco#159] [Q&A] [answered]
目前用的是项目根目录的README.md，不想使用这个文件，因为这个文件中写的内容会呈现在gtihub项目README里面

**Comment by recoluan** [comment]
> 目前用的是项目根目录的README.md，不想使用这个文件，因为这个文件中写的内容会呈现在gtihub项目README里面

这是 vuepress 的设定，没有办法改变，或者把文档放在一个子目录里，这样就不影响根目录的 readme 文件了

**Comment by recoluan** [comment]
试一下这个配可以吗，应该可以解决你的问题：[文档](https://v2.vuepress.vuejs.org/zh/reference/default-theme/config.html#home)

**Reply by LasyIsLazy** [reply]
似乎没有生效。而且这个配置是vuepress默认主题的配置吧

**Reply by recoluan** [reply]
你需要先去指定首页，通过 frontmatter.home = true，再通过 themeConfig.home 来指定这个首页路径。

**Comment by sun0225SUN** [comment, marked as answer]
其实我们可以换一种思路，"修改" GitHub 的 `Readme.md` 位置

https://docs.github.com/zh/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes
> 如果将自述文件放在存储库隐藏的 .github 目录、根目录或 docs 目录中，GitHub 将识别自述文件并自动向存储库访问者显示。
> 如果存储库包含多个自述文件，则按以下顺序从各位置中选择显示的文件：.github 目录，然后是存储库的根目录，最后是 docs 目录。

根据优先级，把GitHub的`Readme.md`写在 `.github` 目录下就可以了，例如：https://github.com/sun0225SUN/hao-docs

**Reply by LasyIsLazy** [reply]
也是一种思路
